pyflakes/test/test_other.py

```diff
@@ -2000,3 +2000,16 @@ def test_raise_notimplemented(self):
         self.flakes('''
         raise NotImplemented
         ''', m.RaiseNotImplemented)
+
+
+class TestAdHocConditions(TestCase):
```

Comment:
What does "AdHocConditions" mean here?

Aren't we just testing the cases where print can be used? Can we come up with a better name please?